lib/tool_belt/redmine/issue.rb

```diff
@@ -15,10 +16,45 @@ def id
       @raw_data['issue']['id']
     end
 
+    def changesets
+      sets = @raw_data['issue']['changesets']
+      return [] if sets.nil?
+      sets
+    end
+
+    # arrange pull request data similar to .changesets
+    def pull_request_commits
+      return [] if pull_requests.nil? || pull_requests.empty?
+
+      client = Octokit::Client.new
```

Comment:
Since you are re-writing this code a bit anyway, want to add in support for GITHUB credentials in case of rate limiting?